src/Testing/InteractsWithEmails.php

```diff
@@ -93,6 +93,19 @@ public function seeEmail($email, $queued = MailGrasp::UNQUEUED)
 
         return $this;
     }
+    
+    public function dontSeeEmail($email, $queued = MailGrasp::UNQUEUED)
```

Comment:
That's cool. Can you add dontSeeEmailInQueue as well? The function can return dontSeeEmail with queue set to true (see seeEmailInQueue).